src/sbt-test/sbt-scalafix/scalafixOnCompileConfig/test

```diff
@@ -0,0 +1,37 @@
+# check if scalafix-on-compile is disabled
+
+> example/compile
+-> example/scalafix
+
+# check if scalafix-on-compile is enabled and uses .scalafix.conf
+
+> set scalafixOnCompile.in(ThisBuild) := true
+-> example/compile
+-> example/scalafix
+
+# check if scalafix-on-compile uses .scalafix-on-compile.conf
+#   and scalafix-explicit-run still uses .scalafix.conf
+
+> set scalafixOnCompileConfig.in(ThisBuild) := file(".scalafix-on-compile.conf"),
```

Comment:
this fails on sbt 0.13.x with the trailing comma
```suggestion
> set scalafixOnCompileConfig.in(ThisBuild) := file(".scalafix-on-compile.conf")
```

Reply:
It was typo. Thank you!